server/app/services/applicant/question/Question.java

```diff
@@ -18,27 +22,71 @@
  *   <li>Present validation errors if any
  * </ul>
  */
-public interface Question {
```

Comment:
Note that all of the additions from this class are moving the implementation from QuestionImpl.java (now deleted) as-is.